packages/expo/CHANGELOG.md

```diff
@@ -10,6 +10,12 @@
 
 ### 💡 Others
 
+## 48.0.17 — 2023-05-11
+
```

Comment:
The version and date will be added automatically by the publish script 😉 
```suggestion
```

Reply:
Thanks 👍